superset-frontend/src/explore/components/controls/MetricControl/AdhocMetricEditPopover/index.jsx

```diff
@@ -272,7 +272,7 @@ export default class AdhocMetricEditPopover extends PureComponent {
   refreshAceEditor() {
     setTimeout(() => {
       if (this.aceEditorRef) {
-        this.aceEditorRef.editor.resize();
+        this.aceEditorRef.editor?.resize();
```

Comment:
If editor is conditional then you should handle resize call as like 
```suggestion
        this.aceEditorRef.editor?.resize?.();
```